### PR TITLE
HackRF One r9

### PR DESCRIPTION
**Pull request by mossmann**
Major changes in this hardware revision:

- change U19 from Si5351C to Si5351A plus external clock distribution
- change U17 from MAX2837 to MAX2839
- add reverse current protection diode to antenna port power supply
- change PCB stack-up
- update design to KiCad 6

### CODE REVIEW

**Comment by mossmann** [comment]
@straithe I would like you to review the block diagrams and other documentation updates.

File: docs/source/hardware_components.rst [hunk 1]
```diff
@@ -26,10 +28,13 @@ Major parts used in HackRF One:
 * `W25Q80BV 8M-bit Flash <https://www.winbond.com/resource-files/w25q80bv%20revk%2020151203.pdf>`__
 
 
+Block Diagrams
```
Comment (straithe) [inline docs/source/hardware_components.rst:31]: Please add subheadings for each of the blog diagrams in this section. My suggestion is that you use "HackRF One R1-R8 Block Diagram" as the first subheading and "HackRF One R9 Block Diagram" as the second sub heading.

**Review by straithe — CHANGES_REQUESTED** [review]
Great work! Just one change requested.

**Review by straithe — APPROVED** [review]
Great job!

**Pull request merged**